terraform/services/ecr-synk-integration/main.tf

```diff
@@ -0,0 +1,99 @@
+
+locals {
+  repos = {
+    ab2d = [
+      "repo:CMSgov/ab2d-bcda-dpc-platform:*",
+      "repo:CMSgov/ab2d-events:*",
+      "repo:CMSgov/ab2d-lambdas:*",
+      "repo:CMSgov/ab2d:*",
+    ]
+    bcda = [
+      "repo:CMSgov/ab2d-bcda-dpc-platform:*",
+      "repo:CMSgov/bcda-app:*",
+    ]
+    dpc = [
+      "repo:CMSgov/ab2d-bcda-dpc-platform:*",
+      "repo:CMSgov/dpc-app:*",
+    ]
+  }
+  admin_app = var.app == "dpc" ? "bcda" : var.app
+}
+
+resource "aws_ssm_parameter" "snyk_integration_role_arn" {
+  name  = "/${var.app}-snyk/role-arn"
+  type  = "String"
+  value = aws_iam_role.snyk.arn
+}
+
+
+data "aws_ssm_parameter" "snyk_integration_role_arn" {
+  name = "/snyk-integration/role-arn"
+}
```

Comment:
It doesn't make sense to both create the SSM parameter and run the data source on the SSM parameter. This is why the parameter is created manually, so we can set the value in AWS and pull it into the terraform.